### PR TITLE
Update documentation about docker image

### PR DESCRIPTION
**Pull request by Shalsh23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,19 +28,18 @@ Current version of the [pipeline](https://github.com/phenoscape/phenoscape-owl-t
 ***
 
 ## Deployment
+1. Download docker image `phenoscape/pipeline-tools` from [here](https://hub.docker.com/r/phenoscape/pipeline-tools).
```
Comment (balhoff) [inline README.md:31]: I don't believe this is necessary. It just happens automatically.

File: README.md [hunk 2]
```diff
@@ -28,19 +28,18 @@ Current version of the [pipeline](https://github.com/phenoscape/phenoscape-owl-t
...
+2. Execute `run.sh` to launch the container. Pass in commands to build different targets as arguments to `run.sh`. Pass `make all` to run the entire pipeline. 
+3. `run.sh` also launches a `bash` shell in the container to interactively work with the pipeline.
```
Comment (balhoff) [inline README.md:33]: I'm not sure this is true. It just runs whatever command you pass as an argument.

File: README.md [hunk 3]
```diff
@@ -28,19 +28,18 @@ Current version of the [pipeline](https://github.com/phenoscape/phenoscape-owl-t
 ***
 
 ## Deployment
+1. Download docker image `phenoscape/pipeline-tools` from [here](https://hub.docker.com/r/phenoscape/pipeline-tools).
+2. Execute `run.sh` to launch the container. Pass in commands to build different targets as arguments to `run.sh`. Pass `make all` to run the entire pipeline.
```
Comment (balhoff) [inline README.md:32]: The way this is phrased makes it sounds like you execute `run.sh` and then pass in commands. Just give a complete working example command line.

File: README.md [hunk 4]
```diff
@@ -28,19 +28,18 @@ Current version of the [pipeline](https://github.com/phenoscape/phenoscape-owl-t
...
-1. [Robot v1.4.0](https://github.com/ontodev/robot)
-2. [Phenoscape OWL tools v1.8](https://github.com/phenoscape/phenoscape-owl-tools) 
-3. [Jena ARQ v3.10](http://jena.apache.org/)
-4. [DOSDP v0.13.1](https://github.com/INCATools/dosdp-tools/releases)
-
-### Usage
-
-Execute `make all` on commandline to build the pipeline.
+```
+docker pull phenoscape/pipeline-tools:v1.0.2
```
Comment (balhoff) [inline README.md:38]: Not necessary, right?

File: README.md (hunk 1 above)
Comment (Shalsh23) [reply README.md:31]: Oh, I didn't know that. That's convenient.

File: README.md (hunk 2 above)
Comment (Shalsh23) [reply README.md:33]: No, I have run this script without passing in any arguments, and it just straight launches the bash shell.

**Comment by Shalsh23** [comment]
> I didn't mean to suggest it needs to be quite so terse. You can give a bit more background, about how the docker container prepackages the tools used within the makefile, you can use it to run the build by using the run.sh as such, etc. Right now it's pretty cryptic. And I don't think that someone trying to build the KB needs to be instructed to change the image version here.

Should I also specify the custom tools that are packaged in the image?

**Comment by balhoff** [comment]
This is good. To list the tools included in the docker container, you could always describe that more fully in the pipeline-tools README, and link there from here. Please open an issue for that and then work on it there.

**Pull request merged**